utils/gnosisClients.ts

```diff
@@ -0,0 +1,12 @@
+import { createPublicClient, createWalletClient, http } from 'viem';
+import { gnosis } from 'wagmi/chains';
+
+export const gnosisPublicClient = createPublicClient({
```

Comment:
why not use the standard client with specifying chain? `useReadContract` and `useWriteContract`